build/dependencies.props

```diff
@@ -29,13 +29,15 @@
     <MicrosoftExtensionsWebEncodersPackageVersion>2.0.0</MicrosoftExtensionsWebEncodersPackageVersion>
     <MicrosoftIdentityModelClientsActiveDirectoryPackageVersion>3.14.1</MicrosoftIdentityModelClientsActiveDirectoryPackageVersion>
     <MicrosoftIdentityModelProtocolsOpenIdConnectPackageVersion>2.1.4</MicrosoftIdentityModelProtocolsOpenIdConnectPackageVersion>
+    <MicrosoftIdentityModelProtocolsWsFederationPackageVersion>5.2.0</MicrosoftIdentityModelProtocolsWsFederationPackageVersion>
```

Comment:
@natemcmaster Does this need to be added to https://github.com/aspnet/Universe/blob/release/2.0/build/dependencies.props?

Reply:
Yup. Chris opened a PR to universe with this.

Reply:
Ah I see there is already a PR for this: https://github.com/aspnet/Universe/pull/844